src/Testing/InteractsWithMail.php

```diff
@@ -147,4 +147,9 @@ protected function seeMessage()
 
         return $this;
     }
+
+    protected function dontSeeMessage()
+    {
+        $this->assertTrue($this->getMessages()->isEmpty());
```

Comment:
Good point, there's a part of me that still wants it because I think people would expect that behavior to possible, if they did need it for whatever reason. 

However, you're correct - right now fluent is not necessary. Thanks for the insight!